alignak/modulesmanager.py

```diff
@@ -117,145 +90,56 @@ def set_max_queue_size(self, max_queue_size):
-            if mod:
-                msg = "Correctly loaded %s as an old-new-style alignak module :|"
-            elif mod_dir:
-                mod = cls.try_very_bad_load(mod_dir)
-                if mod:
-                    msg = "Correctly loaded %s as a very old-style alignak module :s"
-        if msg:
-            logger.info(msg, mod_name)
-        return mod
-
-    def load(self):
-        """Try to import the requested modules ; put the imported modules in self.imported_modules.
-        The previous imported modules, if any, are cleaned before.
-
-        :return: None
+        Try to import the requested modules ; put the imported modules in self.imported_modules.
         """
-        if self.modules_path not in sys.path:
-            sys.path.append(self.modules_path)
-
-        alignak_modules_path = sys.modules['alignak'].__path__[0] + '/modules'
-        if alignak_modules_path not in sys.path:
-            sys.path.append(alignak_modules_path)
-
-        modules_paths = [alignak_modules_path, self.modules_path]
-
-        modules_files = []
-        for path in modules_paths:
-            for fname in listdir(path):
-                if isdir(join(path, fname)):
-                    modules_files.append({'path': path, 'name': fname})
-
-        del self.imported_modules[:]
-        for module in modules_files:
-            mod_file = abspath(join(module['path'], module['name'], 'module.py'))
-            mod_dir = os.path.normpath(os.path.dirname(mod_file))
-            mod = self.try_load(module['name'], mod_dir)
-            if not mod:
-                continue
-            try:
-                is_our_type = self.modules_type in mod.properties['daemons']
-            except Exception as err:
-                logger.warning("Bad module file for %s : cannot check its properties['daemons']"
-                               "attribute : %s", mod_file, err)
-            else:  # We want to keep only the modules of our type
-                if is_our_type:
-                    self.imported_modules.append(mod)
 
         # Now we want to find in theses modules the ones we are looking for
         del self.modules_assoc[:]
-        for mod_conf in self.modules:
-            module_type = uniform_module_type(mod_conf.module_type)
-            for module in self.imported_modules:
-                if uniform_module_type(module.properties['type']) == module_type:
-                    self.modules_assoc.append((mod_conf, module))
-                    break
-            else:  # No module is suitable, we emit a Warning
-                logger.warning("The module type %s for %s was not found in modules!",
-                               module_type, mod_conf.get_name())
+        for mod_conf in mod_confs:
+            try:
+                module = importlib.import_module(mod_conf.module_type)
+                self.find_module_properties_and_get_instance(module, mod_conf.module_type)
```

Comment:
Are you mixing module_type and module name on purpose (see the name variable in find_module_properties_and_get_instace) ?